.circleci/config.yml

```diff
@@ -74,12 +148,23 @@ workflows:
   version: 2
   build:
     jobs:
-      - build:
-          # CircleCI2 will ignore tags without this. https://circleci.com/docs/2.0/workflows/#git-tag-job-execution
+      - compile:
           filters: { tags: { only: /.*/ } }
-      - build-11:
-          # CircleCI2 will ignore tags without this. https://circleci.com/docs/2.0/workflows/#git-tag-job-execution
+
+      - unit-test:
+          requires: [ compile ]
           filters: { tags: { only: /.*/ } }
-      - markdown:
```

Comment:
@ferozco I think we lost this markdown too

Reply:
fixed